pom.xml

```diff
@@ -262,6 +262,7 @@
                 <version>2.21.0</version>
                 <configuration>
                     <argLine>-Xmx2G</argLine>
+					<argLine>-Dfile.encoding=UTF-8</argLine>
```

Comment:
Please indent with spaces rather than tabs, to match the indentation of the rest of the file.

Reply:
Change made.